docs/_documentations/codewind-eclipse-quick-guide.md

```diff
@@ -0,0 +1,167 @@
+
+The Codewind installation includes two parts:
+
+1. The Eclipse plug-in installs when you install Codewind from the [Eclipse Marketplace](https://marketplace.eclipse.org/content/codewind)or when you install by searching in the `Eclipse Extensions` view.
+2. The Codewind backend containers install after you click `Install`. Clicking `Install` downloads the Codewind backend containers, ~1GB. 
+
+### Configuring Codewind to use application stacks
+Configure Codewind to use Appsody templates so you can focus exclusively on your code. These templates include an Eclipse MicroProfile stack that you can use to follow this guide. Complete the following steps to select the Appsody templates:
+
+1. Click the `Codewind` tab. 
+2. Expand `Codewind` by clicking the drop-down arrow.
+3. Right-click `Local [Running]`.
+4. Select `Manage Template Sources...`. 
+5. Select `Appsody Stacks - incubator`.
+6. Click the `OK` button. 
+
+You now have configured Codewind to Appsody templates and can proceed to develop your microservice within Codewind.
+
+If your organization uses customized application stacks and has given you a URL that points to an `index.json` file, you can add it to Codewind: 
+
+1. Return to  `Codewind` and right-click `Local [Running]`. 
+2. Select `Manage Template Sources...`. 
+3. Click the `Add...` button to add your URL.
+4. Add your URL in the `URL:` box in the pop up window and save your changes. 
+
+### Creating an Appsody project
+Appsody helps you develop containerized applications and removes the burden of managing the full software development stack. If you want more context about Appsody, visit the https://appsody.dev/docs[Appsody welcome page]. 
+
+1. Right-click `Local [Running]` under `Codewind` in the `Codewind` tab.
+2. Select `+ Create New Project...`
+    * **Note:** Make sure Docker is running. Otherwise, you get an error. 
+3. Name your project `appsody-calculator`. 
+4. Under `Template`, select `Appsody Open Liberty template`. 
+    * If you don't see an Appsody template, select the `Manage Template Sources...` link at the end of the window.
+    * Select the `Appsody Stacks - appsodyhub` checkbox. 
+    * Click `OK`.
+    * The templates are refreshed, and the Appsody templates are available. 
+5. Click `Finish`.
+    * To monitor your project's progress, right-click on your project, and select `Show Log Files`.
+    * Select `Show All`. Then a `Console` tab is displayed where you see your project's build logs. 
+
+Your project is displayed in the `Local [Running]` section. The progress for creating your project is tracked next to the project's name. 
+
+Your project is complete when you see your project is running and its build is successful. 
+
+### Accessing the application endpoint in a browser
+1. Return to your project under the Codewind tab. 
+2. Right-click your project and select `Open Application`. 
+
+Your application is now opened in the browser, showing the welcome to your Appsody microservice page. 
```

Comment:
`now opened in the browser` => `now opened in a browser ` to match the title of this section.